instrumentation/opentelemetry-instrumentation-botocore/tests/conftest.py

```diff
@@ -61,8 +63,6 @@ def environment():
         os.environ["AWS_ACCESS_KEY_ID"] = "test_aws_access_key_id"
     if not os.getenv("AWS_SECRET_ACCESS_KEY"):
         os.environ["AWS_SECRET_ACCESS_KEY"] = "test_aws_secret_key"
-    if not os.getenv("AWS_SESSION_TOKEN"):
```

Comment:
AFAIK, this is an optional variable - if present, it is used additionally to the others to create a time-sensitive signature, if not present it is not used. Having this here made me unable to regenerate the responses when only passing key id and access key, and I believe removing this should not otherwise cause issues

Reply:
I think I needed because I use temporary credentials, but we'll see when I have to add new recording :)